From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@alkalait__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@alkalait : Thanks for your contribution to the Azure documentation! The author, @hning86, has been notified to review your proposed change.

pablissima commented:
Please merge this

carolinacmoravia committed:
C14396: Asterisk issue for localized versions

Hello, @RodSan,

Localization team has reported source content issue that causes localized version to have different format compared to en-us version.

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

craigshoemaker committed:
Initial draft

craigshoemaker committed:
Pre-test changes

Ja-Dunn committed:
edit pass: service-fabric-hosting-model

Ja-Dunn committed:
edit pass: service-fabric-hosting-model

Ja-Dunn committed:
edit pass: service-fabric-hosting-model

craigshoemaker committed:
Update code samples and add simple server

WilliamAntonRohm committed:
Merge pull request #77 from MicrosoftDocs/master

howdy

Ja-Dunn committed:
edit pass: service-fabric-hosting-model

Ja-Dunn committed:
edit pass: service-fabric-hosting-model

WilliamAntonRohm committed:
Merge pull request #78 from MicrosoftDocs/master

howdy

WilliamAntonRohm committed:
first batch of OMS rebranding files

WilliamAntonRohm committed:
Merge pull request #79 from MicrosoftDocs/master

howdy

kevinvngo committed:
Updates details to Upgrade Process

kevinvngo committed:
Upgrade media

kevinvngo committed:
Uploaded images for upgrade

kevinvngo committed:
Delete empty file

kevinvngo committed:
Delete1

kevinvngo committed:
Delete2

kevinvngo committed:
Renamed images

kevinvngo committed:
Renamed images

kevinvngo committed:
delete1

kevinvngo committed:
delete2

kevinvngo committed:
delete3

kevinvngo committed:
delete image 3

kevinvngo committed:
Delete Upgrade _to_Gen2_3.png

kevinvngo committed:
Add files via upload

kevinvngo committed:
Update upgrade-to-latest-generation.md

kevinvngo committed:
Update upgrade-to-latest-generation.md

kevinvngo committed:
Update upgrade-to-latest-generation.md

kevinvngo committed:
Update upgrade-to-latest-generation.md

carolinacmoravia committed:
L14967: Localizable content blocked in code line

Hello, @davidmu, 

Localization team has reported source content issue that causes localizable text to be blocked from localization.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

craigshoemaker committed:
Updates based off review feedback

carolinacmoravia committed:
L15409: Corrupted character insterted in link reference Line: 36

Hello, @abshamsft, 

Localization team has reported source content issue that causes localized version to have broken format compared to en-us version.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

WilliamAntonRohm committed:
Merge pull request #80 from MicrosoftDocs/master

howdy

craigshoemaker committed:
Update to fine-tune text and new screenshot

kevinvngo committed:
Additional Monitoring Steps

kevinvngo committed:
Bold

carolinacmoravia committed:
M24804: Wrong segmentation

Hello, @rwike77,

Translator has reported possible source content issue.  

Please, help to check my proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR, or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR. 

Many thanks in advance.

WilliamAntonRohm committed:
Merge pull request #81 from MicrosoftDocs/master

hello

dlepow committed:
VM AZ: CLI update

WilliamAntonRohm committed:
reverting 'OMS agent' changes

v-mavick committed:
Migrating devkit-state.md to docs.microsoft.com

craigshoemaker committed:
Add quickstart for JavaScript/HTML

WilliamAntonRohm committed:
OMS rebranding set #4

v-mavick committed:
Correcting issues per Ja-Dunn feedback.

Ja-Dunn committed:
update

Ja-Dunn committed:
fix

craigshoemaker committed:
Add new Next steps link and SAS guidance

gatneil committed:
added missing onte

mimig1 committed:
Cosmos DB: Moving Mimi's files to Sneha & Kelly (10)

douglaslMS committed:
Added new article from PM and TOC entry.

neilpeterson committed:
updated ssh procediure

douglaslMS committed:
Resized three oversized images.

cherylmc committed:
Update vpn-gateway-faq-vnet-vnet-include.md

cherylmc committed:
Update vpn-gateway-table-requirements-include.md

neilpeterson committed:
updated copy and output

ktoliver committed:
fix typo

fix typo

msjuergent committed:
update_caching

v-mavick committed:
Attempting to correct how the Note on line 44 renders.

msjuergent committed:
update_caching2

msjuergent committed:
update_caching3

msjuergent committed:
update_caching4

dlepow committed:
Update az vm create command

Commit:
Update

ktoliver committed:
Merge pull request #37174 from douglaslMS/adf-iterative-dev

Added new article from PM and TOC entry.

PRMerger10 committed:
Merge pull request #37173 from mimig1/04052018meta10

Cosmos DB: Moving Mimi's files to Sneha & Kelly (10)

neilpeterson committed:
output updates

JasonWHowell committed:
Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

PRMerger13 committed:
Merge pull request #37191 from Minewiskan/aas0405

Update

Ja-Dunn committed:
Merge pull request #37096 from v-mavick/devkit-state

Migrating devkit-state.md to docs.microsoft.com

PRMerger19 committed:
Merge pull request #37172 from gatneil/custom_image

added missing onte

PRMerger19 committed:
Merge pull request #37183 from cherylmc/patch-1

Update vpn-gateway-faq-vnet-vnet-include.md

Commit:
Merge pull request #37190 from msjuergent/wadoc

Wadoc

Commit:
Task 1025: Add Always match condition note to Token Auth Denial Code feature

cherylmc committed:
Update vpn-gateway-table-requirements-include.md

cherylmc committed:
Update vpn-gateway-table-requirements-include.md

PRMerger18 committed:
Merge pull request #37193 from JasonWHowell/patch-1

Update stream-analytics-build-an-iot-solution-using-stream-analytics.md

Commit:
Task 1123: Updates for using token auth with query string caching

JasonWHowell committed:
removing unused images

cherylmc committed:
Update vpn-gateway-table-requirements-include.md

neilpeterson committed:
updated image and software installation

jpconnock committed:
updating manager metadata

neilpeterson committed:
acrolinx

PRMerger10 committed:
Merge pull request #37188 from cherylmc/patch-3

Update vpn-gateway-table-requirements-include.md

jpconnock committed:
updating manager metadata

PRMerger15 committed:
Merge pull request #37204 from dksimpson/dean-cdn

Tasks 1123 & 1025: Token auth, rules engine updates

HJToland3 committed:
powershell tweak

craigshoemaker committed:
Fix formatting and update text

tfitzmac committed:
added responses

PRMerger15 committed:
Merge pull request #37213 from HJToland3/master

powershell tweak

tfitzmac committed:
fixes MicrosoftDocs/azure-docs#6660

GitHubber17 committed:
Merge pull request #37207 from jpconnock/jp2

updating manager metadata

GitHubber17 committed:
Merge pull request #37209 from jpconnock/jp3

updating manager metadata

GitHubber17 committed:
Merge pull request #37203 from JasonWHowell/iotRemoveImages

removing unused images

GitHubber17 committed:
Merge pull request #37216 from tfitzmac/0405response

added responses

neilpeterson committed:
copy id_rsa vs copy paste text

neilpeterson committed:
updated copy

neilpeterson committed:
Updated H2

neilpeterson committed:
copy update

ajaycode committed:
ExpressRoute image

dominicbetts committed:
Update ref links

dominicbetts committed:
Fix link

Commit:
Fixed link.

craigshoemaker committed:
Fix metadaat formatting

PRMerger12 committed:
Merge pull request #36015 from carolinacmoravia/patch-654

C14051: Asterisk issue for localized versions

PRMerger16 committed:
Merge pull request #36904 from carolinacmoravia/patch-655

L14967: LOCBLOCKER: Localizable content blocked in code line

PRMerger16 committed:
Merge pull request #37226 from jimdial/patch-130

Fixed link.

craigshoemaker committed:
Update titles

Commit:
Added configure guest OS

@iainfoulds Please sign-off if you're OK with this.  Thanks.

PRMerger20 committed:
Merge pull request #36906 from carolinacmoravia/patch-656

L15409: Corrupted character insterted in link reference Line: 36

Commit:
Added a step.

Added configure guest OS step to create and add sections. @iainfoulds - please sign-off if you're OK with the changes.

Commit:
Added references.

@iainfoulds please sign-off if you're OK with the changes.

Commit:
Fix.

Commit:
Fix.

Commit:
Fix.

Commit:
Fix.

PRMerger14 committed:
Merge pull request #36840 from WilliamAntonRohm/W-oms-00

OMS rebranding -- set #1

Commit:
Added AD tenant info.

Ja-Dunn committed:
fix

Commit:
rewrites/images

Commit:
images

Ja-Dunn committed:
fix

Brenduns committed:
updated image

itechedit committed:
Merge pull request #36579 from craigshoemaker/crs-blob-client-libraries-qs

Quickstart: Upload, list, and delete blobs in Azure Storage using JavaScript client libraries

cephalin committed:
add tutorial in heading

PRMerger14 committed:
Merge pull request #37228 from jimdial/patch-131

Added configure guest OS

PRMerger16 committed:
Merge pull request #37238 from cephalin/umbrella

add tutorial in heading

itechedit committed:
Merge pull request #37018 from kevinvngo/patch-22

Upgrade Doc

cephalin committed:
add links

Brenduns committed:
Adding details from relnotes, which were not a bugged known issue. Also fixing acrolinx score.

v-shils committed:
Merge pull request #37222 from ajaycode/images-for-overview

ExpressRoute image

v-shils committed:
Merge pull request #37182 from neilpeterson/aks-ssh-update

updated ssh procediure

Commit:
Merge pull request #37239 from cephalin/umbrella

add links

Commit:
Merge pull request #37233 from jimdial/patch-134

Added AD tenant info.

ShawnJackson committed:
Merge pull request #36602 from Ja-Dunn/service-fabric-hosting-model

edit pass: service-fabric-hosting-model

Commit:
Merge pull request #37138 from WilliamAntonRohm/W-oms-03b

OMS rebranding -- set #4

Commit:
Added a step.

ktoliver committed:
Merge pull request #37223 from dominicbetts/landing-page-links

Update ref links

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

ktoliver committed:
Merge pull request #37234 from Blackmist/freshness

Freshness

carolinacmoravia committed:
Update service-fabric-sfctl-application.md

itechedit committed:
Merge pull request #37237 from Brenduns/april-02

updated image

PRMerger16 committed:
Merge pull request #37231 from jimdial/patch-133

Added references.

itechedit committed:
Update azure-stack-marketplace.md

PRMerger17 committed:
Merge pull request #37243 from jimdial/patch-136

Added a step.

v-alje committed:
Merge pull request #37246 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger18 committed:
Merge pull request #37230 from jimdial/patch-132

Added a step.

PRMerger18 committed:
Merge pull request #37020 from carolinacmoravia/patch-661

M24804: Wrong segmentation

PRMerger18 committed:
Merge pull request #37192 from dlepow/zone

VM AZ: CLI update

brapel committed:
bing-custom-search: correct azure link

itechedit committed:
Merge pull request #37240 from Brenduns/april-6

Adding details from relnotes, which were not a bugged known issue. Al…

Commit:
Merge pull request #37252 from brapel/bing-custom-search-fix-link

bing-custom-search: correct azure link

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
